jobs/integration/utils.py

```diff
@@ -53,7 +53,7 @@ def timeout_for_current_task(timeout):
     asyncio task will be cancelled, and an asyncio.TimeoutError will be raised.
     """
     loop = asyncio.get_event_loop()
-    task = asyncio.Task.current_task()
+    task = asyncio.current_task()
```

Comment:
[**`asyncio.Task.current_task()`**](https://docs.python.org/3.8/library/asyncio-task.html#asyncio.Task.current_task)  Deprecated in python 3.9